src/mapeo-project.js

```diff
@@ -548,6 +549,100 @@ export class MapeoProject extends TypedEmitter {
   get $icons() {
     return this.#iconApi
   }
+
+  /** @param {Object} opts
+   *  @param {string} opts.configPath
+   */
+  async importConfig({ configPath }) {
+    // check for already present fields and presets and delete them if exist
+    const presets = await this.preset.getMany()
+    if (presets.length !== 0) {
+      for (const preset of presets) {
+        const { fieldIds, versionId: presetVersionId } = preset
+        // delete fields
+        for (const fieldId of fieldIds) {
```

Comment:
Just do the same `this.field.getMany()` and delete them all, rather than this. You could end up trying to delete the same field twice, which will throw an error.

Reply:
I'm wondering about this. If I do smth like:
```js
// this would actually be in parallel
const fields = await this.field.getMany()
for(const {versionId} of fields){
    await this.field.delete(versionId)
}
```
I'm actually not bothering to check if the fields that I'm deleting are referenced by a preset; I'm just deleting every field. Could there be an issue with that? (since we're also deleting every preset I would say there isn't any issue, I'm just double checking...)

Reply:
A loop with an await is always serial. The only way to get parallel async is with promise.all / allSettled. It doesn't actually make much difference for this code because better SQLite is synchronous. But could change in the future, so best to do parallel as best practice.

Reply:
No, I can't see any issue with deleting every field.